src/components/tutorialEditor/stepContent.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState } from "reactn";
+import styled, { AnyStyledComponent } from "styled-components";
+import "../app/normalize.css";
```

Comment:
```suggestion
```
Correct me if I'm wrong but I don't think we need this since we already imported in the top level editor page component.